DOC-2192: 6.7.2 Community docs release. (#2940)
tinymce/tinymce-docs: modules/ROOT/pages/changelog.adoc

NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].

== 6.7.2 - 2023-10-25

> === Fixed
> * Removed use of `async` for editor rendering which caused visual blinking when reloading the editor in-place.
> * Toggling a list that contains an LI element having another list as its first child would remove the remaining content within that LI element.
> * Applying heading formatting to the content of the summary element extended its application to the content of the parent details element.
> 

Author: ```suggestion
=== Fixed
* Applying heading formatting to the content of the `summary` element extended its application to the content of the parent `details` element.
// TINY-10154
* css loading in `toolbardrawertoggletest` flickering.
// TINY-10249
* Toggling a list that contained a list item element — `<li>` — which, in turn, contained another list item element as its first child, removed other content within the first list item element.
// TINY-10213
* If multiple inline formatting options were set at the insertion point, turning any one of these options off turned all the set options off.
//TINY-10132
```

NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].

== 6.7.2 - 2023-10-25

> === Fixed
> * Applying heading formatting to the content of the `summary` element extended its application to the content of the parent `details` element.
> // TINY-10154
> * css loading in `toolbardrawertoggletest` flickering.
> // TINY-10249
> * Toggling a list that contained a list item element — `<li>` — which, in turn, contained another list item element as its first child, removed other content within the first list item element.
> // TINY-10213
> * If multiple inline formatting options were set at the insertion point, turning any one of these options off turned all the set options off.
> //TINY-10132

Author: ```suggestion
=== Fixed
* Applying heading formatting to the content of the `summary` element extended its application to the content of the parent `details` element.
//TINY-10154
* The function `getModifierState` did not work on events passed through the editor as expected.
//TINY-10263
* List items containing a nested list between two text caused some list operations to fail.
//TINY-10268
* css loading in `toolbardrawertoggletest` flickering.
//TINY-10249
* Toggling a list that contained a list item element — `<li>` — which, in turn, contained another list item element as its first child, removed other content within the first list item element.
//TINY-10213
* If multiple inline formatting options were set at the insertion point, turning any one of these options off turned all the set options off.
//TINY-10132
```

NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].

== 6.7.2 - 2023-10-25

=== Fixed

> * Applying heading formatting to the content of the `summary` element extended its application to the content of the parent `details` element.

Author: ```suggestion
```

* The function `getModifierState` did not work on events passed through the editor as expected.

> * List items containing a nested list between two text caused some list operations to fail.

Author: ```suggestion
* Indenting or outdenting a list item that contained non list item siblings after it would result in those siblings being removed.
```

* Removed use of async for editor rendering which caused visual blinking when reloading the editor in-place.

> * Indenting or outdenting a list item that contained non list item siblings after it would result in those siblings being removed.

Author: ```suggestion
* Toggling a list that contained a list item element — <li> — which, in turn, contained another list item element as its first child, removed other content within the first list item element.
```

NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].

== 6.7.2 - 2023-10-25

=== Fixed
* Applying heading formatting to the content of the `summary` element extended its application to the content of the parent `details` element.
* The function `getModifierState` did not work on events passed through the editor as expected.
* List items containing a nested list between two text caused some list operations to fail.
* Removed use of async for editor rendering which caused visual blinking when reloading the editor in-place.
* Indenting or outdenting a list item that contained non list item siblings after it would result in those siblings being removed.

> * If multiple inline formatting options were set at the insertion point, turning any one of these options off turned all the set options off.

Author: @ArvinJ-H Can you confirm that is has been moved to 6.8, so i can remove from 6.7.2 community release.
```suggestion
```

NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].

== 6.7.2 - 2023-10-25

=== Fixed
* Applying heading formatting to the content of the `summary` element extended its application to the content of the parent `details` element.

> //TINY-10154

Author: ```suggestion
```

* The function `getModifierState` did not work on events passed through the editor as expected.

> //TINY-10263

Author: ```suggestion
```

* List items containing a nested list between two text caused some list operations to fail.

> //TINY-10268

Author: ```suggestion
```

> * css loading in `toolbardrawertoggletest` flickering.

Reviewer: `toolbardrawertoggletest` what is that? It should likely state that the theme loading is no longer async as the changelog item in the tinymce repo? So maybe `Reduced UI flickering when reloading the editor by making the theme initialization sync instead of async`

Author: ```suggestion
* Removed use of async for editor rendering which caused visual blinking when reloading the editor in-place.
```

Author: @spocke update the below, thanks for the catch

> //TINY-10249

Author: ```suggestion
```

> * Toggling a list that contained a list item element — `<li>` — which, in turn, contained another list item element as its first child, removed other content within the first list item element.

Reviewer: Hard to describe that weird scenario:
```suggestion
* Indenting or outdenting a list item that had non list item siblings after it would remove those siblings.
```

Author: ```suggestion
* Indenting or outdenting a list item that contained a non list item siblings after it would result in those siblings being removed.
```

Author: @spocke Ive updated the copy edit suggestion, thoughts?

Reviewer: ```suggestion
* Indenting or outdenting a list item that contained non list item siblings after it would result in those siblings being removed.
```

> //TINY-10213

Author: ```suggestion
```

* If multiple inline formatting options were set at the insertion point, turning any one of these options off turned all the set options off.
//TINY-10132

> 

Author: Note: remove TINY-ticket ref before publishing.

> //TINY-10132

Author: ```suggestion
```
